primitives/evm/Cargo.toml

```diff
@@ -19,10 +19,15 @@ serde = { version = "1.0.101", optional = true, features = ["derive"] }
 codec = { package = "parity-scale-codec", version = "2.0.0", default-features = false }
 evm = { version = "0.25.0", default-features = false, features = ["with-codec"] }
 impl-trait-for-tuples = "0.1"
+hex = { version = "0.4.0", optional = true }
```

Comment:
If you can, please make those dev dependencies.

Reply:
I think the problem I run into here is that `#[cfg(test)]` only seems to work within a single crate. In other words:

```rust
// inside primitives/evm/src/lib.rs
#[cfg(test)]
pub use precompile::test_precompile_consensus_tests;
```

doesn't export `test_precompile_consensus_tests` when running `cargo test` inside e.g. `frame/evm/precompile/blake2`.

I'm open to suggestions on reorganizing this (or being taught that I'm wrong!)

One solution would be to move this test function to its own crate. I'd be happy to do this, though it strikes me as adding complexity.

Reply:
> One solution would be to move this test function to its own crate. I'd be happy to do this, though it strikes me as adding complexity.

I would really prefer this!